management/azure_mgmt_network/spec/2017-09-01/load_balancers_spec.rb

```diff
@@ -70,7 +70,7 @@
 
     load_balancing_rule = LoadBalancingRule.new
     load_balancing_rule.name = load_balancing_rule_name
-    frontend_ip_configuration_sub_resource = MsRestAzure::SubResource.new
```

Comment:
this change is not related to the PR, right? I believe it's related to https://github.com/Azure/azure-sdk-for-ruby/pull/1106, where should have this been caught when the other PR changes happened?